### PR TITLE
Add an email layout and example email

### PR DESCRIPTION
**Pull request by fofr**
Uses a new email layout.
Set the `subject`, `from` and `to` variables. 

Screenshot 2021-12-04 at 12 28 20

### CODE REVIEW

**Review by paulrobertlloyd — APPROVED** [review]
Fixed a CSS listing error and a stray `'` in a template. But absolutely love this!

**Comment by benilovj** [comment]
@fofr it'd be really handy if the email example could cover [all the types of elements that Notify supports](https://www.notifications.service.gov.uk/using-notify/guidance/edit-and-format-messages) – I've just got the styling wrong when trying to add a bulleted list...

**Comment by fofr** [comment]
> Small suggestion: it'd be really handy if the email example could cover [all the types of elements that Notify supports](https://www.notifications.service.gov.uk/using-notify/guidance/edit-and-format-messages) – I've just got the styling wrong when trying to add a bulleted list...

Added in ba98ab9

File: app/views/examples/email.njk [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <li>horizontal rules</li>
+  </ul>
+
+  <p>They cannot have subheadings or bold text.</p>
+
+  <p class="govuk-inset-text">
+    <a href="https://www.notifications.service.gov.uk/using-notify/guidance/edit-and-format-messages">Read the Notify documentation</a>
+  </p>
+
+  <p>GOV.UK</p>
```
Comment (paulrobertlloyd) [inline app/views/examples/email.njk:29]: This now reads a bit odd. I’d consider removing it.

```suggestion
```

**Pull request merged**